### ISSUE TITLE
Add license to code

### ISSUE THREAD

**Issue by thiteixeira** [codeforboston/voiceapp311#132]
Add MIT software license to the codebase.
Reference: https://help.github.com/articles/adding-a-license-to-a-repository/

**Comment by jmartini** [comment]
@thiteixeira is this done?

**Comment by Lou-Boston** [comment]
No. I was told I didn’t have the account privilege to add the licenses so I
was assigned something else.

Lou Madge

**Comment by thiteixeira** [comment]
Yes, the license is added to the repository
